objects/_modal-popover.scss

```diff
@@ -94,3 +94,9 @@ $popover-shadow: 0 0 15px -3px rgba(0,0,0,0.35);
     }
 }
 
+.modal--popover--dark {
```

Comment:
we already have variation named `--dk`, so TBH I'm not sure about `--dark` 🤔

Reply:
yea, the `--dk` one isn't very dark, I think it was initially and then they changed it. Calling your new one one dark seems right, maybe we rename `--dk`? `--gray`? `--medium`?

Reply:
I merged it, we can address the `--dk` naming separately.

Reply:
Yeah, good call. I'll create a PR renaming `--dk` to `--gray`